OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -37,18 +37,29 @@
 
 # Global Variables
 PackagesDirectory = 'packages'
-BundleFileName = 'omsagent-1.4.0-45.universal.x64.sh'
-HUtilObject = None
+BundleFileName = 'omsagent-1.4.1-45.universal.x64.sh'
+GUIDRegex = r'[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}'
+GUIDOnlyRegex = r'^' + GUIDRegex + '$'
+
+# Paths
+OmsAdminPath = '/opt/microsoft/omsagent/bin/omsadmin.sh'
+OmsAgentServiceScript = '/opt/microsoft/omsagent/bin/service_control'
+OMIConfigEditorPath = '/opt/omi/bin/omiconfigeditor'
```

Comment:
Please have the global variable naming convention consistent. Either be "OMSAdminPath" or change "OMIConfigEditorPath" be "OmiConfigEditorPath".

Reply:
Done